airflow/utils/operator_helpers.py

```diff
@@ -164,10 +164,7 @@ def determine(
 
     def unpacking(self) -> Mapping[str, Any]:
         """Dump the kwargs mapping to unpack with ``**`` in a function call."""
-        # Context is a TypedDict at lint time, and Mypy would complain it cannot
-        # be used in isinstance. But the call works at runtime since the type is
-        # actually implemented as a custom mapping, so we ignore the Mypy error.
-        if self._wildcard and isinstance(self._kwargs, Context):  # type: ignore
+        if self._wildcard and isinstance(self._kwargs, Context):
```

Comment:
Despite mypy complaining about this before _in this PR_ it's now not, at least not for me when I run `pre-commit run -a mypy` locally.

I want to see what number CI gets -- it should be 102 errors left for the first mypy job.

Reply:
Confirmed it's fine now. I have _no_ idea what was going on before but this was def a problem.

Reply:
Possibly other fix fixed it. Also while MyPy is stable and repeatable, it is somewhat "vulnerable" to small changes - it generates more or less loosely related errors after small changes that should not (at least obviously) trigger them - at least that's the impression I had sometimes.